custom_components/garmin_connect/__init__.py

```
@@ -528,6 +540,24 @@ async def _async_update_data(self) -> dict:
             _LOGGER.debug(
                 "Error occurred while processing HRV summary status data")
 
+        # Blood pressure — extract the most recent measurement from today's data.
+        # The API returns measurementSummaries[].measurements[] ordered newest-first.
+        try:
+            if bp_data and "measurementSummaries" in bp_data:
+                for bp_summary in bp_data["measurementSummaries"]:
+                    if "measurements" in bp_summary and bp_summary["measurements"]:
```

Ruff (0.15.9)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)